website/docs/addon_deadline_admin.md

```diff
@@ -16,121 +16,368 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Introduction
 
-## Preparation
+AYON integration for Deadline primarily uses the Deadline web service to send jobs.
 
-For [AWS Thinkbox Deadline](https://www.awsthinkbox.com/deadline) support you need to set a few things up in both AYON and Deadline itself
+The integration consists of two parts:
+- The Deadline custom plugins.
+- A `GlobalJobPreLoad` Deadline Script (triggered for each Deadline job submitted via the Deadline addon).
 
-1. Deploy AYON executable to all nodes of Deadline farm. See [Install & Run](admin_launcher_distribute.md)
+The `GlobalJobPreLoad` script manages the population of submitted jobs with the correct environment variables, using settings from the AYON Deadline Plug-in.
 
-2. Create a new AYON Bundle with Deadline Addon version.
+Additionally, the addon streamlines Unreal Deadline submissions by utilizing the version control addon. See [Unreal Engine 5 Plug-in](#unreal-engine-5-plug-in).
 
-3. Set up *Deadline Web API service*. For more details on how to do it, see [here](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/web-service.html).
 
-4. Provide values for all Deadline servers you want to use in `ayon+settings://deadline`
+## Preparation
 
-![Webservice url](assets/deadline_webserver_config.png)
+For [AWS Thinkbox Deadline](https://www.awsthinkbox.com/deadline) support you need to set a few things up in both AYON and Deadline itself
```

Comment:
This is accidentally duplicated?

Reply:
thank you
https://github.com/ynput/ayon-documentation/pull/301/commits/bb80cb4c580de335fc21eb4bb24fedadfecd20af